.github/workflows/danger.yml

```diff
@@ -1,22 +0,0 @@
-name: Danger actions
-'on':
-  pull_request:
-    types: [opened, edited, reopened, synchronize, assigned, unassigned, review_requested, review_request_removed]
-jobs:
-  danger-ci:
-    name: Danger CI
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@master
-      - uses: actions/setup-node@master
-        with:
-          node-version: 12.x
-        env:
-          RUNNER_TEMP: /tmp
-      - name: yarn install
-        run: yarn install
-      - name: Danger CI
-        uses: vtex/danger@master
-        env:
-          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
-          REQUIRE_CHANGELOG_VERSION: false
```

Comment:
These workflows seem useful, why removing them?

Reply:
DangerJS has never been set as a pattern in VTEX and the tests made with it brought more questions and problems than solutions.

Reply:
I see your point! What about the build/test workflows? It occurs to me that we could replace them with a Drone example, but that's out of the scope of this PR